packages/xhost.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Xhost < Package
+	description 'Server access control program for X'
```

Comment:
Are these tabs? I can't exactly tell on this, but that's what they look like. Could you change them to two spaces?

Reply:
You know, here's the thing about this. So I originally used vim as my only text editor and had it set up how I like it. It would convert my tabs to spaces and make everything pretty. Then uberhacker updated the vim package and broke all my stuff, but that's when I had VSCode working so it was no big deal because I had that set up how I like it and it would do the same thing with tabs to spaces... BUT now everyone broke both vim and VSCode so I had to resort to using Caret and it's a little frustrating. So basically I will convert these, I just wanted to explain to you why my life is the way it is right now.

Reply:
OK. I use `nano`, but it's a tad simple for "expert" users. :)